tests/baselines/reference/accessorDeclarationEmitVisibilityErrors.errors.txt

```diff
@@ -0,0 +1,12 @@
+tests/cases/compiler/accessorDeclarationEmitVisibilityErrors.ts(2,18): error TS2304: Cannot find name 'DoesNotExist'.
+tests/cases/compiler/accessorDeclarationEmitVisibilityErrors.ts(2,18): error TS4106: Parameter 'arg' of accessor has or is using private name 'DoesNotExist'.
+
+
+==== tests/cases/compiler/accessorDeclarationEmitVisibilityErrors.ts (2 errors) ====
+    export class Q {
+        set bet(arg: DoesNotExist) {}
+                     ~~~~~~~~~~~~

```

Comment:
names that don't exist create a "is using private name" errors? ok..

Reply:
Eeeeyup.